go-controller/pkg/util/util.go

```diff
@@ -329,40 +331,52 @@ func IsClusterIP(svcVIP string) bool {
 	return false
 }
 
-// GetActiveNetworkForNamespace returns the active network for the given namespace
-// based on the NADs present in that namespace.
+func defaultNetConf() *ovncnitypes.NetConf {
+	return &ovncnitypes.NetConf{NetConf: cnitypes.NetConf{Name: types.DefaultNetworkName}}
+}
+
+// FindActiveNetworkForNamespace returns the active network for the given namespace
```

Comment:
why is it find and not get? because we changed semantics to nil? "get" and "set" is what we usually use..
btw its a bit confusing pointer to Netconf struct is coming in the end

suggestion: "FindActiveNetworkForNamespace returns the NetConf struct of the active network for the given namespace based on the NADs present in that namespace."

Reply:
> why is it find and not get? because we changed semantics to nil? "get" and "set" is what we usually use.. btw its a bit confusing pointer to Netconf struct is coming in the end
> 

As commented before we are looking at a database and it's a costly operation so is better to use `find`.

> suggestion: "FindActiveNetworkForNamespace returns the NetConf struct of the active network for the given namespace based on the NADs present in that namespace."

ack